### ISSUE TITLE
Compact mode, right sidebar takes up too much space

### ISSUE THREAD

**Issue by castaway** [sindresorhus/caprine#1485]
My install (AppImage on Linux) auto-updated a few days ago, and now in compact mode (aka about the side of an 80-char wide xterm), the main chat window contents disappear slightly under the enormous right-sidebar (which I have little use for)..

Would it be possible to allow the user to resize (aka drag&size) the right sidebar smaller, or even turn it off altogether? I can't find a setting for that.

.. In settings the "sidebar" options (eg "hide sidebar") do nothing (now, I think they used to) - which sidebar are they for?

To doublecheck it wasnt me, I also fetched the new 2.51.2 AppImage manually.. still the same :(

**Comment by mattnoir** [comment]
there have been major changes in messenger itself, making caprine basically without functionality. Try to look for other issues here to gain more info and you can verify your new messenger look by trying some keyboard shortcuts or toggling dark mode on and off. If nothing works, it's new messenger and we have to wait

**Comment by castaway** [comment]
:(, thanks for the update!

**Comment by bonzupii** [comment]
It's definitely new messenger that has left Caprine almost completely without functionality.

**Comment by dusansimic** [comment]
I have already reported that issue in #1436. It might not be related to the new messenger design since @castaway is talking about right side bar in compact mode which. Take a look at that issue and tell me if you have the same problem.

**Comment by castaway** [comment]
> I have already reported that issue in #1436. It might not be related to the new messenger design since @castaway is talking about right side bar in compact mode which. Take a look at that issue and tell me if you have the same problem.

Yup I saw yours, which is similar.. I'd like to just get rid of the thing tho, mostly :) I interpreted #1436 as wanting it to be wider .. If you meant "adjust size on demand", then yup, same thing

**Comment by castaway** [comment]
... I just accidentally found the "Conversation information" button which removes it... was that always there!?

**Comment by dusansimic** [comment]
Yes it was, it was in both earlier version of UI and the newer one. If this fixes your issue then this should be closed.

**Comment by castaway** [comment]
Certainly helps, not sure why the update reset it.. Would be nicer if the thing could be manually resized though, or does your issue cover that?

**Comment by dusansimic** [comment]
Issue I reported doesn't recommend resizing, rather making sidebar full width in compact mode. I personally find it better to just make it full width in compact mode rather than resize it manually to the size that is comfortable. When these issues get reviewed by maintainers, we'll know which one will be implemented.
